src/test/java/com/cegeka/academy/service/UserChallengeServiceTest.java

```diff
@@ -140,5 +145,13 @@ public void testGetChallengesByUserIdWithNoResult(){
 
     }
 
+    @Test
+    public void testRateUserByOwner() {
```

Comment:
Aici m-as folosi putin de expected si actual. E destul de greu de urmarit assert-ul in forma asta.